### DISCUSSION TITLE
Investiny Install

### DISCUSSION THREAD

**Discussion by F-Hisham** [alvarobartt/investiny#77] [Q&A] [answered]
Hi all,

Since this morning I'm trying to install investiny (as investpy doesn't work for the moment) in pycharm but it doesn't work. 
I tried to install it via:
- anaconda -> package not found and not available in anaconda
- pip install investiny --> Installed but not recognised in the code and does not appear in the python interpreter packages
- Python interpreter packages in pycharm ... Fail to install

Do youknow how to process to install properly investiny, ideally make it available in anaconda as well

Many thanks in advance for your help!

**Comment by alvarobartt** [comment, marked as answer]
Hi @F-Hisham I think `investiny` is not working either, as Investing.com doesn't want to actively support it and they included Cloudflare V2 protection in their APIs, so probably won't work either...

Anyway, which Python version are you using? `investiny` is just available for Python 3.8 or above!

**Comment by F-Hisham** [comment]
Hi Alvaro,

Thank you for your quick reply and for the information.

I'm currently using the version 3.10 as interpreter in PyCharm, when I check the version via the cmd (python -V), it indicates that I have the version 3.9.12.

It's a shame! Investpy were very usefull to quickly get the data required, so the only solution remaining is to use a scraper?
